Singularity/Singularity/Platform/PlatformActionState.cs

```diff
@@ -8,7 +8,7 @@ public enum PlatformActionState
         /// <summary> Available: meaning it can be executed,
         /// the button for the action can be clicked </summary>
         Available,
-        /// <summary> Active: meaning it is currently being executed,
+        /// <summary> ActiveWindow: meaning it is currently being executed,
```

Comment:
you change the comment tag to ActiveWindow but the enum is still just active

Reply:
oh, this was automatically changed - my bad. I'll fix it